llvm/utils/git/code-lint-helper.py

```diff
@@ -289,8 +291,65 @@ def _clean_clang_tidy_output(self, output: str) -> str:
         return ""
 
 
+class Doc8LintHelper(LintHelper):
+    name: Final = "doc8"
+    friendly_name: Final = "RST documentation linter"
 
-ALL_LINTERS = (ClangTidyLintHelper(),)
+    def instructions(self, files_to_lint: Iterable[str], args: LintArgs) -> str:
+        return f"doc8 -q {' '.join(files_to_lint)}"
+
+    def filter_changed_files(self, changed_files: Sequence[str]) -> Sequence[str]:
+        filtered_files: List[str] = []
+        for filepath in changed_files:
+            _, ext = os.path.splitext(filepath)
```

Comment:
Will be good idea to put all three conditions into `_should_lint_file` and use `filter(self._should_lint_file, changed_files)` in loop.